DashKitUI/Views/NotificationSettingsView.swift

```diff
@@ -40,29 +40,44 @@ struct NotificationSettingsView: View {
                 footer: LocalizedString("The App notifies you in advance of Pod expiration.  Set the number of hours advance notice you would like to have.", comment: "Footer text for omnipod reminders section")
             ) {
                 expirationReminderRow
-                if let scheduledReminderDate = scheduledReminderDate  {
+            }
+
+            if let scheduledReminderDate = scheduledReminderDate, let allowedDates = allowedScheduledReminderDates {
+                RoundedCard(
+                    footer: LocalizedString("This is a reminder that you scheduled when you paired your current Pod.", comment: "Footer text for scheduled reminder area"))
+                {
+                    Text("Scheduled Reminder")
                     Divider()
-                    scheduledReminderRow(scheduledDate: scheduledReminderDate)
+                    scheduledReminderRow(scheduledDate: scheduledReminderDate, allowedDates: allowedDates)
                 }
             }
 
             RoundedCard(footer: LocalizedString("The App notifies you when the amount of insulin in the Pod reaches this level.", comment: "Footer text for low reservoir value row")) {
                 lowReservoirValueRow
             }
 
-            RoundedCardTitle(LocalizedString("Critical Alerts", comment: "Title for critical alerts description"))
-                .padding(.bottom, 1)
-            RoundedCardFooter(LocalizedString("The reminders above will not sound if your device is in Silent or Do Not Disturb mode.\n\nThere are other critical Pod alerts and alarms that will sound even if you device is set to Silent or Do Not Disturb mode.", comment: "Description text for critical alerts"))
+            VStack(alignment: .leading) {
+                RoundedCardTitle(LocalizedString("Critical Alerts", comment: "Title for critical alerts description"))
+                    .padding(.bottom, 1)
+                RoundedCardFooter(LocalizedString("The reminders above will not sound if your device is in Silent or Do Not Disturb mode.\n\nThere are other critical Pod alerts and alarms that will sound even if you device is set to Silent or Do Not Disturb mode.", comment: "Description text for critical alerts"))
+            }
+            .padding(.horizontal, 16)
         }
         .navigationBarTitle(LocalizedString("Notification Settings", comment: "navigation title for notification settings"))
     }
     
+    var expirationDefaultFormatter = QuantityFormatter(for: .hour())
+    
+    var expirationDefaultString: String {
+        return expirationDefaultFormatter.string(from: HKQuantity(unit: .hour(), doubleValue: Double(expirationReminderDefault)), for: .hour())!
```

Comment:
[nit] Using HealthKit to format a duration seems odd.

Reply:
Hour units translations should be handled by the system.